gyp/common.py

```diff
@@ -604,3 +605,18 @@ def CrossCompileRequested():
           os.environ.get('AR_target') or
           os.environ.get('CC_target') or
           os.environ.get('CXX_target'))
+
+
+def GetStdout(cmdlist, with_stderr=False):
+  """
+  Returns the content of standard output returned by invoking |cmdlist|.
+  Raises |GypError| if the command return with a non-zero return code.
+  """
+  job = subprocess.Popen(cmdlist, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+  out, err = job.communicate()
+  if job.returncode != 0:
+    if with_stderr:
+      print(out, file=sys.stderr)
+      print(err, file=sys.stderr)
```

Comment:
Should __both__ of these lines be stderr?

Reply:
Removed this whole function